azure-iot-device/tests/common/test_asyncio_compat.py

```diff
@@ -7,8 +7,11 @@
 import pytest
 import asyncio
 import sys
+import logging
 from azure.iot.device.common import asyncio_compat
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
>INFO [](start = 34, length = 4)

do we want INFO or DEBUG?  I think we have a ton of logging at INFO but almost none at DEBUG, so it doesn't matter much today, but do we want to shift more to DEBUG and make INFO less noisy?